apps/events/templates/events/events_list.html

```diff
@@ -0,0 +1,54 @@
+{% extends "subpage.html" %}
+
+{% load crispy_forms_tags %}
+
+{% block title %}Events{% endblock title%}
+
+{% block content %}
+
+<div class="container">
+    <div class="w-75 mx-auto">
+        {% if perms.cosmos.change_events %}
+        <div class="float-end">
+            <a class="btn btn-cosmos-primary" href="{% url 'events:events-create' %}">
+                <i class="bi bi-plus-lg"></i>
+                Add new Event
+            </a>
+        </div>
+        {% endif %}
+        <h2>Events</h2>
+        <div class="row row-cols-1 row-cols-md-3 g-2 mt-2" id="EventGrid">
+        {% for event in events_list %}
+            <div class="col">
+                <div class="card h-100">
+                    {# transform is added so that stretched link does not go into footer #}
+                    <img class="card-img-top events-image" src="{{ event.image.url }}">
+                    <div class="card-body">
+                        <h5 class="card-title">{{ event.name }}</h5>
+                        <p class="event-truncate">{{ event.lead }}</p>
+                        <a class="stretched-link" href="{% url 'events:events-view' event.id %}"></a>
+                    </div>
+                    <div class="card-footer">
+                    {% comment %} Temp solution to fix card height {% endcomment %}
+                        {% comment %} {% if perms.cosmos.change_event and perms.cosmos.delete_event %}
```

Comment:
Why are these permissions commented out exactly?

Reply:
Because right now there is no solution to making the whole card clickable except the modify/delete buttons without the card height being screwed up.
I'll look into a fix for this